### DISCUSSION TITLE
Application Request Form

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/data-tools#22] [Ideas]
@usamabilal  just a creating a discussion to record the following slack message. So it doesn't get lost on Slack. Thought this would be an appropriate repository until we formalize things later on. 

>new project RA for SALURBAL (octavio) can help us with developing a potential form for app requests, etc. We should at some point convene to think about how to move that forward to help systematize future app requests

**Comment by usamabilal** [comment]
fantastic, lets chat sometime later this month
